documentation/src/main/asciidoc/topics/ref_xsite_jgroups_relay_custom.adoc

```diff
@@ -24,9 +24,41 @@
 
 [source,xml,options="nowrap",subs=attributes+]
 ----
-include::config_examples/jgroups-relay.xml[]
+<config xmlns="urn:org:jgroups"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/jgroups-4.1.xsd">
+
+    <!-- Use TCP for inter-cluster transport. -->
+    <TCP bind_addr="127.0.0.1"
+         bind_port="7200"
+         port_range="30"
+
+         thread_pool.min_threads="0"
+         thread_pool.max_threads="8"
+         thread_pool.keep_alive_time="5000"
+    />
+
+    <!-- Use TCPPING for inter-cluster discovery. -->
+    <TCPPING timeout="3000"
+             initial_hosts="127.0.0.1[7200]"
+             port_range="3"
+             ergonomics="false"/>
+
+    <jgroups>
```

Comment:
this is not a valid JGroups' configuration file. 
What are you trying to achieve here?

Reply:
@wfink please clarify ^